layouts/partials/community_links.html

```diff
@@ -2,28 +2,31 @@
 
 <section class="row td-box td-box--4 td-box--gradient td-box--height-auto linkbox">
 <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
-<h2>Learn and Connect</h2>		
-<p>Using or want to use {{ .Site.Title }}? Find out more here:
```

Comment:
You are removing  all the shortcodes but why ??? You can give input  to the short codes from  atsign.dev

Reply:
Compare it to https://devstaging.atsign.wtf/community/ 

The Site title is currently blank in the toml file.

But if it is filled with anything it the short code shows up in the top right corner of the home page as well.

so, I hard coded it.

Reply:
@ platform is not going to change for the time being.